sdk/communication/azure-communication-callautomation/src/test/resources/session-records/CallAutomationAsyncClientLiveTests.createPSTNCallAndHangupTest[1].json

```diff
@@ -0,0 +1,107 @@
+{
+  "networkCallRecords" : [ {
+    "Method" : "POST",
+    "Uri" : "https://REDACTED.communication.azure.com/identities?api-version=2022-06-01",
+    "Headers" : {
+      "User-Agent" : "azsdk-java-azure-communication-identity/1.3.1 (18.0.2.1; Windows 10; 10.0)",
+      "x-ms-client-request-id" : "8a9d95a6-b5c0-48e5-88f8-657076d13f1a",
+      "Content-Type" : "application/json"
+    },
+    "Response" : {
+      "X-Cache" : "CONFIG_NOCACHE",
+      "content-length" : "101",
+      "api-supported-versions" : "2020-07-20-preview2, 2021-02-22-preview1, 2021-03-07, 2021-10-31-preview, 2021-11-01, 2022-06-01, 2022-10-01",
+      "retry-after" : "0",
+      "StatusCode" : "201",
+      "Date" : "Fri, 23 Sep 2022 17:42:22 GMT",
+      "Strict-Transport-Security" : "max-age=2592000",
+      "X-Processing-Time" : "80ms",
+      "MS-CV" : "gTGYudsk+U2FmmIS+d0QVw.0",
+      "X-Azure-Ref" : "0/u8tYwAAAADGzwDnYV9vQKA80+9fv5s3Q0hHRURHRTE1MDYAOWZjN2I1MTktYThjYy00Zjg5LTkzNWUtYzkxNDhhZTA5ZTgx",
+      "Body" : "{\"identity\":{\"id\":\"8:acs:816df1ca-971b-44d7-b8b1-8fba90748500_00000014-0b53-1f54-99bf-a43a0d009143\"}}",
+      "x-ms-client-request-id" : "8a9d95a6-b5c0-48e5-88f8-657076d13f1a",
+      "Content-Type" : "application/json; charset=utf-8",
+      "Request-Context" : "appId="
+    },
+    "Exception" : null
+  }, {
+    "Method" : "POST",
+    "Uri" : "https://REDACTED.communication.azure.com/calling/callConnections?api-version=2022-04-07-preview",
+    "Headers" : {
+      "User-Agent" : "azsdk-java-azure-communication-callautomation/1.0.0-beta.1 (18.0.2.1; Windows 10; 10.0)",
+      "x-ms-client-request-id" : "6b33f89a-95fa-4863-8183-64369a5c8e53",
+      "Content-Type" : "application/json"
+    },
+    "Response" : {
+      "X-Cache" : "CONFIG_NOCACHE",
+      "content-length" : "576",
+      "retry-after" : "0",
+      "X-Azure-Ref" : "0/u8tYwAAAADGmnT7zsrtS6dyvg/qszGhQ0hHRURHRTE1MDYAOWZjN2I1MTktYThjYy00Zjg5LTkzNWUtYzkxNDhhZTA5ZTgx",
+      "StatusCode" : "201",
+      "X-Microsoft-Skype-Chain-ID" : "99b9d62d-3013-47d2-8dd8-73fb745390b7",
+      "Body" : "{\"callConnectionId\":\"811f6a00-d2c9-4139-86ff-e46cbdde30d9\",\"source\":{\"callerId\":{\"value\":\"+18337597849\"},\"identifier\":{\"rawId\":\"8:acs:816df1ca-971b-44d7-b8b1-8fba90748500_00000014-0b53-1f54-99bf-a43a0d009143\",\"kind\":\"communicationUser\",\"communicationUser\":{\"id\":\"8:acs:816df1ca-971b-44d7-b8b1-8fba90748500_00000014-0b53-1f54-99bf-a43a0d009143\"}}},\"targets\":[{\"rawId\":\"4:+16476282217\",\"kind\":\"phoneNumber\",\"phoneNumber\":{\"value\":\"+16476282217\"}}],\"callConnectionState\":\"connecting\",\"callbackUri\":\"https://localhost\",\"mediaSubscriptionId\":\"72e11d73-b9af-4e40-8993-be50c260ff09\"}",
```

Comment:
can the phone numbers and acs ids be sanitized?

Reply:
Updated. I have marked them as REDACTED.